drivers/scheduler/k8s/specs/aut-postgres/aut-postgres-storage.yaml

```diff
@@ -0,0 +1,26 @@
+##### Portworx storage class
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+    name: px-postgres-sc
+provisioner: kubernetes.io/portworx-volume
+parameters:
+   repl: "3"
+   priority_io: "high"
+   shared: "true"
```

Comment:
postgres should not use a shared volume. If both pods run on the same node (due to the pod affinity), PX will allow mounting a regular volume on multiple pods